main.c

```diff
@@ -178,6 +180,29 @@ static int prepare_for_sleep(sd_bus_message *msg, void *userdata,
 
 	return 0;
 }
+static int prepare_for_lock(sd_bus_message *msg, void *userdata,
```

Comment:
Should probably renamed those to `handle_lock`

Reply:
I'll rename all three while i'm at it

Reply:
Well `prepare_for_sleep` really handles the `PrepareForSleep` event, so it'd make sense to keep it as is.